BambooTracker/BambooTracker.pro

```diff
@@ -391,4 +390,38 @@ RESOURCES += \
 TRANSLATIONS += \
     res/lang/bamboo_tracker_fr.ts
 
+equals(QT_MAJOR_VERSION, 5) {
+    lessThan(QT_MINOR_VERSION, 12) {
+        message(using a workaround for missing 'lrelease' option in Qt <5.12...)
+        
+        for(tsfile, TRANSLATIONS) {
+          qmfile   = $$tsfile
+          qmfile  ~= s/.ts$/.qm/
+          qmlang   = $$tsfile
+          qmlang  ~= s,res/lang/bamboo_tracker_,,
+          qmlang  ~= s,.ts$,,
+          
+            thisqm = $${qmfile}.commands
+          $$thisqm = lrelease $$tsfile
+          
+          QMAKE_EXTRA_TARGETS  += $${qmfile}
+          
```

Comment:
To make sure this will run on the default target. (rebuilds translation on command `make`)
```
          PRE_TARGETDEPS += $${qmfile}
```

Reply:
Added, see https://github.com/rerrahkr/BambooTracker/pull/61/commits/ee82d87f261e20d6248ae21984704e22819cecd6.